pages/reset.tsx

```diff
@@ -0,0 +1,247 @@
+                mb={5}
+            />
+            {/* check to make sure passwords match at this point 
+            verify that is original password changes, both checks still run 
+            */}
+            <PasswordInput
+                name="password"
+                label="Re-Enter your Password"
+                placeholder="Password"
+                isInvalid={passwordInvalid.isInvalid}
+                errorMessage={passwordInvalid.reason}
+                isRequired
+                onChange={(event) => {
+                    setConfirmPassword(event.currentTarget.value);
+                }}
+                onBlur={() => {
+                    if (confirmPassword != newPassword) {
+                        setPasswordInvalid({
+                            isInvalid: true,
+                            reason: "Passwords Do Not Match",
+                        });
+                        setCanContinue(false);
+                    } else {
+                        setPasswordInvalid({
+                            isInvalid: false,
+                            reason: "",
+                        });
+                        setCanContinue(true);
+                    }
+                }}
+                mb={5}
+            />
+            <Text textAlign="left">
+                Passwords must contain one uppercase, one lowercase, one number,
+                one symbol and must be at least 8 characters.
+            </Text>
+            <AuthButton
+                text="Reset Password"
+                onClick={incrementStage}
+                isDisabled={!canContinue}
+            />
+        </>
+    );
+
+    const stages = [
+        {
+            title: "Reset Password",
+            component: EmailStage,
+        },
+        {
+            title: "Check Email",
+            component: (
+                <Box>
+                    <Text>
+                        Password recovery instructions have been sent to your
+                        email.
+                    </Text>
+                    <AuthButton
+                        text="Open Email App"
+                        onClick={() => (location.href = "maito:l")}
+                        isDisabled={false}
+                    />
+                    <Center>
+                        <Text mt="5" as="u">
+                            <Link onClick={incrementStage} mt="5">
+                                Continue
+                            </Link>
+                        </Text>
+                    </Center>
+                    <Text mt="70%">
+                        Did not recieve the email? Check your spam folder or{" "}
+                        <Link onClick={decrementStage}>
+                            <Text as="u">try again.</Text>
+                            {/* Verify whether try again takes users back or will initiate "send email again" flow */}
+                        </Link>
+                    </Text>
```

Comment:
should this be positioned at the lower/on the bottom of the screen?

Reply:
Yup- this is positioned at the bottom of the screen